service/service.go

```diff
@@ -234,8 +234,23 @@ func (svc *Service) Start(wg *sync.WaitGroup, c chan os.Signal) {
 						log.Info("Payment Details Response : ",
 							log.Data{keys.PaymentDetails: paymentDetails, keys.StatusCode: statusCode})
 
-						//Filter accepted payments from GovPay
-						if paymentDetails.PaymentStatus == "accepted" {
+						if isRefundTransaction(pp) {
+							refund := data.RefundResource{}
+							for _, ref := range paymentResponse.Refunds {
+								if ref.RefundId == pp.RefundId {
+									refund = ref
+								}
+							}
+
+							if refund.Status == "success" {
```

Comment:
what about non-successful refunds? We should add a log if we ever encounter these for whatever reason